demo/demoproject/views.py

```diff
@@ -9,6 +9,15 @@
 from chartjs.views.lines import BaseLineChartView, HighchartPlotLineChartView
 from chartjs.views.pie import HighChartPieView, HighChartDonutView
 
+import os
+import sys
+import importlib
+_parent = os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
+sys.path.append(os.path.join(_parent, 'chartjs'))
+util = importlib.import_module('util')
```

Comment:
I had trouble with this import. I would have liked to say `from chartjs.util import date_range, value_or_null` etc., but that wasn't working for some reason and I resorted to this.

Reply:
You should keep `from chartjs.util import date_range, value_or_null` and `pip install -e django-chartjs` in the virtualenv you are working on so that you can install the working version of the lib.